syntax = "proto3";
package sv_rcon;
option optimize_for = LITE_RUNTIME;

enum response_t
{
	SERVERDATA_RESPONSE_VALUE       = 0;
	SERVERDATA_RESPONSE_UPDATE      = 1;
	SERVERDATA_RESPONSE_AUTH        = 2;
	SERVERDATA_RESPONSE_CONSOLE_LOG = 3;
	SERVERDATA_RESPONSE_STRING      = 4;
	SERVERDATA_RESPONSE_REMOTEBUG   = 5;
}

message response
{
    optional int32      responseID   = 1;
    optional response_t responseType = 2;
    optional string     responseBuf  = 3;
    optional string     responseVal  = 4;
}
